docs/Background-Unity.md

```diff
@@ -0,0 +1,77 @@
+# Background: Unity
```

Comment:
I think Kozzy's comment about the CogSci and ML backgrounds applies here too. There's lots of great places to learn about Unity, and we shouldn't try to replicate that here. 

In general documentation should be as succinct as possible while still telling users what they need to know. I think here that mean keeping the AnimalAI docs focused on what AnimalAI adds to what already exists, and perhaps providing external links where necessary.